plutus-tx-plugin/test/Budget/9.6/sumAtIndices.uplc.golden

```diff
@@ -0,0 +1,14 @@
+(program
+   1.1.0
+   (\d ->
+      (\nt ->
+         (\s ->
+            (\nt ->
+               (\s ->
+                  (\s -> addInteger (addInteger s s) s)
+                    (unIData (force headList (force tailList nt))))
+                 (unIData (force headList nt)))
+              ((\tail -> tail (tail (force tailList nt)))
+                 (\eta -> force tailList eta)))
+           (unIData (force headList nt)))
+        (force tailList (unListData d))))
```

Comment:
That's a lot of forcing, why not bind `force tailList` and `force headList` to a variable? Or do you expect CSE to clean it up? Particularly when you already seem to do that in `(\eta -> force tailList eta)`. Why the eta-expansion btw?

Reply:
We don't CSE work-free terms such as `force tailList`. As mentioned in the CSE Note, doing so may reduce size but always increases cost, and CSE currently favors smaller cost. I don't think there is a way to manually bind `force tailList` to a variable, is there?

Reply:
It's not really work-free, since both `Force ` and `Builtin` have a cost. `Var` has a cost too, but it's 1 against 2, so binding it to a variable should be an overall cost reduction.

> I don't think there is a way to manually bind force tailList to a variable, is there?

I don't know.